### ISSUE TITLE
aria-checked inconsistencies

### ISSUE THREAD

**Issue by christophe-g** [material-components/material-components-web-components#1913]
`aria-checked` should be implemented for `mwc-radio`, `mwc-checkbox` and `mwc-switch`.

The current status for each is: 
- radio: not implemented
- switch: not implemented (*EDIT*: ooups, it is actually implemented in @material/switch/foundation.js, but so far away from the template)
- checkbox: not correct. `aria-checked` is correctly set to `mixed` when checked state is indeterminate. However true checked state is not setting `aria-checked` to true. 

I can provide a couple of PR to fix that. However, I need some kind of green light to avoid committing time to code that will not be integrated. 

For the time being, as this is needed pretty soon on my end, I will locally override the render method for each of those components (and post here if anyone else is facing the same problem).

**Comment by taylor-vann** [comment]
Hey @christophe-g, thanks for submitting and we appreciate the concern.

We vet our "aria-checked" attributes and are fairly confident they are correct and function as intended.

I will be closing this bug but feel free to open a new bug if you find this is not true. Also, if you could, please provide a JS-Bin or a specific example of code. It helps us help each other!

Thank you

**Comment by christophe-g** [comment]
> It helps us help each other!

This really is the spirit of this humble contribution. 

However my time is also valuable, and I do not want to re-submit a new issue. please re-open this one if whatever below is convincing enough. 

## checkbox: 
- open https://material-components.github.io/material-components-web-components/demos/checkbox/
- check the first checkbox
- input shall have `aria-checked` attribute set to true. It does not:
```html
<input type="checkbox" class="mdc-checkbox__native-control" data-indeterminate="false" value="">
```

Responsible code for this behavior : https://github.com/material-components/material-components-web-components/blob/147ca002180c55ae7bab6dc2a7a1184f9d05e8a5/packages/checkbox/mwc-checkbox-base.ts#L144 

It should be: `const ariaChecked = this.indeterminate ? 'mixed' : this.checked ? 'true' : 'false';`

## Radio
- please do the same with https://material-components.github.io/material-components-web-components/demos/radio/

no `aria-checked` implementation (nor that I can see in radio foundation), I do not see test for `aria-checked` either.

## Switch
As said above handling on `aria-checked` is done on switch foundation. It would be much clearer (and coherent with how it is handled for mwc-checkbox) to have it directly on the render function.
